src/server/handlers/manifest-handler.ts

```diff
@@ -9,15 +9,15 @@ let manifest: Awaited<ReturnType<typeof generateManifestJson>> | undefined =
   undefined;
 
 export default async (req: Request, res: Response) => {
-  if (!manifest || manifest.start_url !== getHttpBaseExternal()) {
+  if (!manifest) {
     const headers = setForwardedHeaders(req.headers);
```

Comment:
Maybe we should do away with this also? The forwardedHeaders includes auth, and might make this uncacheable.

Reply:
"This" being the `if` statement? The entire function?